src/puppet.ts

```diff
@@ -754,11 +754,11 @@ export abstract class Puppet extends EventEmitter {
   public abstract async messageFile (messageId: string) : Promise<FileBox>
   public abstract async messageUrl (messageId: string)  : Promise<UrlLinkPayload>
 
-  public abstract async messageForward (receiver: Receiver, messageId: string)              : Promise<void>
-  public abstract async messageSendText (receiver: Receiver, text: string)                  : Promise<void>
-  public abstract async messageSendContact (receiver: Receiver, contactId: string)          : Promise<void>
-  public abstract async messageSendFile (receiver: Receiver, file: FileBox)                 : Promise<void>
-  public abstract async messageSendUrl (receiver: Receiver, urlLinkPayload: UrlLinkPayload) : Promise<void>
+  public abstract async messageForward (receiver: Receiver, messageId: string)                    : Promise<void>
+  public abstract async messageSendText (receiver: Receiver, text: string, atUserList?: string[]) : Promise<void>
```

Comment:
Please follow the same naming style as the original code: `atUserList` should be renamed to `mentionIdList` or `mentionContactIdList`.

Because If we use the different name for the same thing, it will cause confusion for the future developers (which will including ourselves)

Reply:
Sure, will do

Reply:
Updated